### ISSUE TITLE
[Paper Note] NVIDIA Nemotron 3: Efficient and Open Intelligence, NVIDIA+, arXiv'25, 2025.12

### ISSUE THREAD

**Issue by AkihikoWatanabe** [AkihikoWatanabe/paper_notes#3965]
# URL
- preprint: https://arxiv.org/abs/2512.20856
- blog: https://developer.nvidia.com/blog/inside-nvidia-nemotron-3-techniques-tools-and-data-that-make-it-efficient-and-accurate/?ncid=so-twit-776109

# Authors
  - NVIDIA
  - :
  - Aaron Blakeman
  - Aaron Grattafiori
  - Aarti Basant
  - Abhibha Gupta
  - Abhinav Khattar
  - Adi Renduchintala
  - Aditya Vavre
  - Akanksha Shukla
  - Akhiad Bercovich
  - Aleksander Ficek
  - Aleksandr Shaposhnikov
  - Alex Kondratenko
  - Alexander Bukharin
  - Alexandre Milesi
  - Ali Taghibakhshi
  - Alisa Liu
  - Amelia Barton
  - Ameya Sunil Mahabaleshwarkar
  - Amir Klein
  - Amit Zuker
  - Amnon Geifman
  - Amy Shen
  - Anahita Bhiwandiwalla
  - Andrew Tao
  - Anjulie Agrusa
  - Ankur Verma
  - Ann Guan
  - Anubhav Mandarwal
  - Arham Mehta
  - Ashwath Aithal
  - Ashwin Poojary
  - Asif Ahamed
  - Asit Mishra
  - Asma Kuriparambil Thekkumpate
  - Ayush Dattagupta
  - Banghua Zhu
  - Bardiya Sadeghi
  - Barnaby Simkin
  - Ben Lanir
  - Benedikt Schifferer
  - Besmira Nushi
  - Bilal Kartal
  - Bita Darvish Rouhani
  - Boris Ginsburg
  - Brandon Norick
  - Brandon Soubasis
  - Branislav Kisacanin
  - Brian Yu
  - Bryan Catanzaro
  - Carlo del Mundo
  - Chantal Hwang
  - Charles Wang
  - Cheng-Ping Hsieh
  - Chenghao Zhang
  - Chenhan Yu
  - Chetan Mungekar
  - Chintan Patel
  - Chris Alexiuk
  - Christopher Parisien
  - Collin Neale
  - Cyril Meurillon
  - Damon Mosk-Aoyama
  - Dan Su
  - Dane Corneil
  - Daniel Afrimi
  - Daniel Lo
  - Daniel Rohrer
  - Daniel Serebrenik
  - Daria Gitman
  - Daria Levy
  - Darko Stosic
  - David Mosallanezhad
  - Deepak Narayanan
  - Dhruv Nathawani
  - Dima Rekesh
  - Dina Yared
  - Divyanshu Kakwani
  - Dong Ahn
  - Duncan Riach
  - Dusan Stosic
  - Edgar Minasyan
  - Edward Lin
  - Eileen Long
  - Eileen Peters Long
  - Elad Segal
  - Elena Lantz
  - Ellie Evans
  - Elliott Ning
  - Eric Chung
  - Eric Harper
  - Eric Tramel
  - Erick Galinkin
  - Erik Pounds
  - Evan Briones
  - Evelina Bakhturina
  - Evgeny Tsykunov
  - Faisal Ladhak
  - Fay Wang
  - Fei Jia
  - Felipe Soares
  - Feng Chen
  - Ferenc Galko
  - Frank Sun
  - Frankie Siino
  - Gal Hubara Agam
  - Ganesh Ajjanagadde
  - Gantavya Bhatt
  - Gargi Prasad
  - George Armstrong
  - Gerald Shen
  - Gorkem Batmaz
  - Grigor Nalbandyan
  - Haifeng Qian
  - Harsh Sharma
  - Hayley Ross
  - Helen Ngo
  - Herbert Hum
  - Herman Sahota
  - Hexin Wang
  - Himanshu Soni
  - Hiren Upadhyay
  - Huizi Mao
  - Huy C Nguyen
  - Huy Q Nguyen
  - Iain Cunningham
  - Ido Galil
  - Ido Shahaf
  - Igor Gitman
  - Ilya Loshchilov
  - Itamar Schen
  - Itay Levy
  - Ivan Moshkov
  - Izik Golan
  - Izzy Putterman
  - Jan Kautz
  - Jane Polak Scowcroft
  - Jared Casper
  - Jatin Mitra
  - Jeffrey Glick
  - Jenny Chen
  - Jesse Oliver
  - Jian Zhang
  - Jiaqi Zeng
  - Jie Lou
  - Jimmy Zhang
  - Jinhang Choi
  - Jining Huang
  - Joey Conway
  - Joey Guman
  - John Kamalu
  - Johnny Greco
  - Jonathan Cohen
  - Joseph Jennings
  - Joyjit Daw
  - Julien Veron Vialard
  - Junkeun Yi
  - Jupinder Parmar
  - Kai Xu
  - Kan Zhu
  - Kari Briski
  - Katherine Cheung
  - Katherine Luna
  - Keith Wyss
  - Keshav Santhanam
  - Kevin Shih
  - Kezhi Kong
  - Khushi Bhardwaj
  - Kirthi Shankar
  - Krishna C. Puvvada
  - Krzysztof Pawelec
  - Kumar Anik
  - Lawrence McAfee
  - Laya Sleiman
  - Leon Derczynski
  - Li Ding
  - Lizzie Wei
  - Lucas Liebenwein
  - Luis Vega
  - Maanu Grover
  - Maarten Van Segbroeck
  - Maer Rodrigues de Melo
  - Mahdi Nazemi
  - Makesh Narsimhan Sreedhar
  - Manoj Kilaru
  - Maor Ashkenazi
  - Marc Romeijn
  - Marcin Chochowski
  - Mark Cai
  - Markus Kliegl
  - Maryam Moosaei
  - Matt Kulka
  - Matvei Novikov
  - Mehrzad Samadi
  - Melissa Corpuz
  - Mengru Wang
  - Meredith Price
  - Michael Andersch
  - Michael Boone
  - Michael Evans
  - Miguel Martinez
  - Mikail Khona
  - Mike Chrzanowski
  - Minseok Lee
  - Mohammad Dabbah
  - Mohammad Shoeybi
  - Mostofa Patwary
  - Nabin Mulepati
  - Najeeb Nabwani
  - Natalie Hereth
  - Nave Assaf
  - Negar Habibi
  - Neta Zmora
  - Netanel Haber
  - Nicola Sessions
  - Nidhi Bhatia
  - Nikhil Jukar
  - Nikki Pope
  - Nikolai Ludwig
  - Nima Tajbakhsh
  - Nir Ailon
  - Nirmal Juluru
  - Nishant Sharma
  - Oleksii Hrinchuk
  - Oleksii Kuchaiev
  - Olivier Delalleau
  - Oluwatobi Olabiyi
  - Omer Ullman Argov
  - Omri Puny
  - Oren Tropp
  - Ouye Xie
  - Parth Chadha
  - Pasha Shamis
  - Paul Gibbons
  - Pavlo Molchanov
  - Pawel Morkisz
  - Peter Dykas
  - Peter Jin
  - Pinky Xu
  - Piotr Januszewski
  - Pranav Prashant Thombre
  - Prasoon Varshney
  - Pritam Gundecha
  - Przemek Tredak
  - Qing Miao
  - Qiyu Wan
  - Rabeeh Karimi Mahabadi
  - Rachit Garg
  - Ran El-Yaniv
  - Ran Zilberstein
  - Rasoul Shafipour
  - Rich Harang
  - Rick Izzo
  - Rima Shahbazyan
  - Rishabh Garg
  - Ritika Borkar
  - Ritu Gala
  - Riyad Islam
  - Robert Hesse
  - Roger Waleffe
  - Rohit Watve
  - Roi Koren
  - Ruoxi Zhang
  - Russell Hewett
  - Russell J. Hewett
  - Ryan Prenger
  - Ryan Timbrook
  - Sadegh Mahdavi
  - Sahil Modi
  - Samuel Kriman
  - Sangkug Lim
  - Sanjay Kariyappa
  - Sanjeev Satheesh
  - Saori Kaji
  - Satish Pasumarthi
  - Saurav Muralidharan
  - Sean Narentharen
  - Sean Narenthiran
  - Seonmyeong Bak
  - Sergey Kashirsky
  - Seth Poulos
  - Shahar Mor
  - Shanmugam Ramasamy
  - Shantanu Acharya
  - Shaona Ghosh
  - Sharath Turuvekere Sreenivas
  - Shelby Thomas
  - Shiqing Fan
  - Shreya Gopal
  - Shrimai Prabhumoye
  - Shubham Pachori
  - Shubham Toshniwal
  - Shuoyang Ding
  - Siddharth Singh
  - Simeng Sun
  - Smita Ithape
  - Somshubra Majumdar
  - Soumye Singhal
  - Stas Sergienko
  - Stefania Alborghetti
  - Stephen Ge
  - Sugam Dipak Devare
  - Sumeet Kumar Barua
  - Suseella Panguluri
  - Suyog Gupta
  - Sweta Priyadarshi
  - Syeda Nahida Akter
  - Tan Bui
  - Teodor-Dumitru Ene
  - Terry Kong
  - Thanh Do
  - Tijmen Blankevoort
  - Tim Moon
  - Tom Balough
  - Tomer Asida
  - Tomer Bar Natan
  - Tomer Ronen
  - Tugrul Konuk
  - Twinkle Vashishth
  - Udi Karpas
  - Ushnish De
  - Vahid Noorozi
  - Vahid Noroozi
  - Venkat Srinivasan
  - Venmugil Elango
  - Victor Cui
  - Vijay Korthikanti
  - Vinay Rao
  - Vitaly Kurin
  - Vitaly Lavrukhin
  - Vladimir Anisimov
  - Wanli Jiang
  - Wasi Uddin Ahmad
  - Wei Du
  - Wei Ping
  - Wenfei Zhou
  - Will Jennings
  - William Zhang
  - Wojciech Prazuch
  - Xiaowei Ren
  - Yashaswi Karnati
  - Yejin Choi
  - Yev Meyer
  - Yi-Fu Wu
  - Yian Zhang
  - Yigong Qin
  - Ying Lin
  - Yonatan Geifman
  - Yonggan Fu
  - Yoshi Subara
  - Yoshi Suhara
  - Yubo Gao
  - Zach Moshe
  - Zhen Dong
  - Zhongbo Zhu
  - Zihan Liu
  - Zijia Chen
  - Zijie Yan
# Abstract
  - We introduce the Nemotron 3 family of models - Nano, Super, and Ultra. These models deliver strong agentic, reasoning, and conversational capabilities. The Nemotron 3 family uses a Mixture-of-Experts hybrid Mamba-Transformer architecture to provide best-in-class throughput and context lengths of up to 1M tokens. Super and Ultra models are trained with NVFP4 and incorporate LatentMoE, a novel approach that improves model quality. The two larger models also include MTP layers for faster text generation. All Nemotron 3 models are post-trained using multi-environment reinforcement learning enabling reasoning, multi-step tool use, and support granular reasoning budget control. Nano, the smallest model, outperforms comparable models in accuracy while remaining extremely cost-efficient for inference. Super is optimized for collaborative agents and high-volume workloads such as IT ticket automation. Ultra, the largest model, provides state-of-the-art accuracy and reasoning performance. Nano is released together with its technical report and this white paper, while Super and Ultra will follow in the coming months. We will openly release the model weights, pre- and post-training software, recipes, and all data for which we hold redistribution rights.
# Translation (by gpt-4o-mini)
- Nemotron 3ファミリーのモデル - Nano、Super、Ultraを紹介します。これらのモデルは、強力なエージェント機能、推論能力、会話能力を提供します。Nemotron 3ファミリーは、Mixture-of-ExpertsハイブリッドMamba-Transformerアーキテクチャを使用して、業界最高のスループットと最大1Mトークンのコンテキスト長を実現しています。SuperおよびUltraモデルはNVFP4でトレーニングされ、モデルの品質を向上させる新しいアプローチであるLatentMoEを組み込んでいます。さらに、2つの大きなモデルには、テキスト生成を高速化するためのMTPレイヤーも含まれています。すべてのNemotron 3モデルは、マルチ環境強化学習を使用してポストトレーニングされており、推論、マルチステップツールの使用、細かな推論予算の制御をサポートします。最小のモデルであるNanoは、推論において非常にコスト効率が高いまま、同等のモデルに対して精度で優れています。Superは、コラボレーティブエージェントやITチケット自動化などの高ボリュームのワークロードに最適化されています。最大のモデルであるUltraは、最先端の精度と推論性能を提供します。Nanoは、その技術報告書とこのホワイトペーパーと共にリリースされ、SuperとUltraは今後数ヶ月以内にリリースされる予定です。モデルの重み、トレーニング前後のソフトウェア、レシピ、および再配布権を持つすべてのデータはオープンにリリースされる予定です。
# Summary (by gpt-4o-mini)
- Nemotron 3ファミリーのモデル（Nano、Super、Ultra）は、強力なエージェント機能と推論能力を提供し、Mixture-of-ExpertsハイブリッドMamba-Transformerアーキテクチャを採用。SuperとUltraはLatentMoEを組み込み、MTPレイヤーでテキスト生成を高速化。全モデルはマルチ環境強化学習でポストトレーニングされ、Nanoはコスト効率が高く、Superは高ボリュームワークロードに最適化、Ultraは最先端の精度を提供。モデルの重みやデータはオープンにリリース予定。

**Comment by AkihikoWatanabe** [comment]
元ポスト: https://x.com/NVIDIAAIDev/status/2000587035166343344?s=20

**Comment by AkihikoWatanabe** [comment]
解説: https://x.com/hillbig/status/2001063481207046484?s=20

**Comment by AkihikoWatanabe** [comment]
Artificial Intelligenceによるポイント解説&ベンチマーキング:
https://x.com/ArtificialAnlys/status/2000602570092675402?s=20

**Comment by AkihikoWatanabe** [comment]
所見:
https://x.com/percyliang/status/2000608134205985169?s=20

training data, RL environment, training codeも含めて公開されているとのこと。

**Comment by AkihikoWatanabe** [comment]
ポイント解説: https://x.com/iScienceLuvr/status/2000570256566493590?s=20

**Comment by AkihikoWatanabe** [comment]
所見:https://x.com/theturingpost/status/2001087448299065372?s=46&t=Y6UuIHB0Lv0IpmFAjlc2-Q
